CHANGES.rst

```diff
@@ -5,23 +5,7 @@ Please note:
 the change log will only get updated after first release - for now please use the
 `commit log <https://github.com/letsencrypt/letsencrypt/commits/master>`_.
 
+To see the changes in a given release, inspect the github milestone for the
```

Comment:
Hah, I wanted to trigger a travis rerun of this branch without erasing the previous failure.  I intended make a one character change to a consequence-free file, but ended up realising the file needed love :)